scripts/commands/bump-version.js

```diff
@@ -35,7 +35,7 @@ async function bumpVersion() {
         return error( 'Aborting version bump.' );
     }
 
-    if ( ! await createNewBranch( `update-version-${ newVersion }` ) ){
+    if ( ! await createNewBranch( `release/version-${ newVersion }` ) ){
```

Comment:
I believe we need to use a format of `release/x.y.z` to match the checks for the smoke test automation:
https://github.com/Automattic/wc-calypso-bridge/blob/7ce77010b9af030537d7629ac5bb68b394adc472/.github/workflows/ci-release.yml#L25
```suggestion
    if ( ! await createNewBranch( `release/${ newVersion }` ) ){
```

Reply:
Ah, you're correct! Thank you!